Engineering/Backend/Testing Made Simple Best Practices for Golang Test.md

```diff
@@ -0,0 +1,234 @@
+# Testing Made Simple: Best Practices for Golang Testing Success
```

Comment:
Should start with formatter data

![image](https://github.com/dwarvesf/brain/assets/12707960/8aa58b11-3c6b-4ebf-8485-4113b85e1449)